modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -17,7 +17,7 @@ const addMenuItems = (editor: Editor, selectionTargets: SelectionTargets) => {
 
   const insertTableAction = ({ numRows, numColumns }) => {
     editor.undoManager.transact(function () {
-      InsertTable.insert(editor, numColumns, numRows);
+      InsertTable.insert(editor, numColumns, numRows, 0, 0);
```

Comment:
actual feature things - this is for the table grid so until we add header rows/cols to the widget, this can be hardcoded to 0